src/elements/hx-error/index.spec.js

```diff
@@ -0,0 +1,38 @@
+import { fixture, expect } from '@open-wc/testing';
+
+/**
+ * <hx-error> component tests
+ *
+ * @type HXErrorElement
+ *
+ */
+describe('<hx-error> tests', () => {
```

Comment:
Please add a screenshot of the test run.

Reply:
Changes have been done, please review.